### PR TITLE
Update staking.md

### PR DESCRIPTION
**Pull request by michelle-aptos**
added section on how commission is distributed

### CODE REVIEW

File: developer-docs-site/docs/concepts/staking.md [hunk 1]
```diff
@@ -195,7 +197,14 @@ When your lockup period expires, it will be automatically renewed, so that you c
 
 ### Unlocking your stake
 
-You can request to unlock your stake at any time. However, your stake will only become withdrawable when your current lockup expires. This can be at most as long as the fixed lockup duration. 
+You can request to unlock your stake at any time. However, your stake will only become withdrawable when your current lockup expires. This can be at most as long as the fixed lockup duration. You will continue earning rewards on your stake until it becomes withdrawable. 
+
+The principal amount is updated when any of the following actions occur:
+1. Operator[requests commission unlock](../nodes/validator-node/operator/staking-pool-operations.md#requesting-commission)
```
Comment (xindingw) [inline developer-docs-site/docs/concepts/staking.md:203]: Maybe Add a space after "Operator".

File: developer-docs-site/docs/concepts/staking.md [hunk 2]
```diff
@@ -195,7 +197,14 @@ When your lockup period expires, it will be automatically renewed, so that you c
...
+2. Staker (owner) withdraws funds
+3. Staker (owner) switches operators
+
+When the staker unlocks stake, this also triggers a commission unlock. The full commission amount for any staking rewards earned is unlocked. This is not proportional to the unlock stake amount. Commission is distributed to the operator when `request commission` is called a second time or when staker withdraws (distributes) the unlocked stake.
```
Comment (xindingw) [inline developer-docs-site/docs/concepts/staking.md:207]: Commission can be distributed only when it has already been inactive(need wait until the current lockup period ends).

Comment (michelle-aptos) [reply developer-docs-site/docs/concepts/staking.md]: clarified: Commission is distributed to the operator after the lockup ends when `request commission` is called a second time or when staker withdraws (distributes) the unlocked stake

File: developer-docs-site/docs/concepts/staking.md [hunk 3]
```diff
@@ -263,6 +272,7 @@ Note that rewards are given only to the **leader-validators**, i.e., validators
 All the validator rewards are also subject to lockup period as they are added to the original staked amount. 
 :::
 
+
```
Comment (xindingw) [inline developer-docs-site/docs/concepts/staking.md:275]: Why do we need this empty line?

**Review by xindingw — APPROVED** [review]

**Pull request merged**